### ISSUE TITLE
Flickering when playing H.264 video exported from PowerPoint with DXVA2 or D3D11

### ISSUE THREAD

**Issue by josiasmat** [Nevcairiel/LAVFilters#339]
I exported a video in MP4 format (H.264 codec) from Microsoft PowerPoint 2019, and when I played it using Media Player Classic/LAV video decoder, the video flickers during the slide transitions.

This issue only occurs when using DXVA2 or D3D11 acceleration. When using Intel QuickSync or software decoding, flickering does not occur. Using the native decoder from Windows also works without issues.

Using LAV Filters version 0.74.1.34-git, from K-Lite Codec Pack. Sample video file attached.

[pwptest.zip](https://github.com/Nevcairiel/LAVFilters/files/4431657/pwptest.zip)

**Comment by sbuethe** [comment]
I just saw this issue here. It is exactly what I am seeing with PowerPoint video exports. I just posted a very similar message on doom9 two days ago.

I tested the PowerPoint exports in MPC-BE (1.5.6 and 1.5.7), MPC-HC (1.9.10) with DXVA cb or native, D3D11 selected. As mentioned above, software decoding works as expected.

A simple demo file as well as a screen capture of the problem on my computer is available here:
https://www.sendspace.com/filegroup/...OfJamwiq22IVnA

It also happens in VLC player, but it will run fine using e. g. ffplay.exe (maybe because of software decoding).

It happens on Intel UHD 620 and on nVidia RTX 2080 with up to date drivers, but not if QuickSync is selected on the UHD 620 and CUVID on the RTX 2080.

Very annoying these days as there is a lot of pre-recorded PowerPoint videos being presented on virtual conferences.

Many thanks for taking a look at it.
